packages/core/content-manager/shared/contracts/history-versions.ts

```diff
@@ -0,0 +1,14 @@
+import type { Entity, UID } from '@strapi/types';
+
+/**
+ * Unlike other Content Manager contracts, history versions can't be created via the API,
```

Comment:
When you say the API I guess you mean the content-api? I mean could someone still create a history version using documentService or db.query?

Reply:
Ah I meant something different, that it's not created by a dedicated `POST /history-versions`, instead they're created as part of another action